### PR TITLE
[Website] update use-cases example to the website

### PR DESCRIPTION
**Pull request by mchoi8739**
*Description of changes:*

- Update readthedoc website to add use-cases notebooks: https://github.com/aws/amazon-sagemaker-examples/tree/master/use-cases
- Most of the changes are proper spacing and setting up index.rst files for rendering.

**Doc build tested on my local machine.**
The CI will fail since these are sequential notebooks with storemagic. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: use-cases/customer_churn/0_cust_churn_overview_dw.ipynb [hunk 1]
```diff
@@ -4,16 +4,7 @@
    "cell_type": "markdown",
```
Comment (jkroll-aws) [inline use-cases/customer_churn/0_cust_churn_overview_dw.ipynb:4]: <p>Nit: trailing underscore - <code>Service_</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: use-cases/predmaint/1_dataprep_dw_job_predmaint.ipynb [hunk 2]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```
Comment (jkroll-aws) [inline use-cases/predmaint/1_dataprep_dw_job_predmaint.ipynb:4]: <p>Inconsistency - <code>DataWrangler</code>  vs <code>Data Wrangler</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: use-cases/customer_churn/0_cust_churn_overview_dw.ipynb (hunk 1 above)
Comment (mchoi8739) [reply use-cases/customer_churn/0_cust_churn_overview_dw.ipynb:4]: fixed

File: use-cases/predmaint/1_dataprep_dw_job_predmaint.ipynb (hunk 2 above)
Comment (mchoi8739) [reply use-cases/predmaint/1_dataprep_dw_job_predmaint.ipynb:4]: fixed

**Review by jkroll-aws — APPROVED** [review]

**Pull request merged**